librz/analysis/p/analysis_bf.c

```diff
@@ -7,35 +8,446 @@
 #include <rz_asm.h>
 #include <rz_analysis.h>
 
-static size_t countChar(const ut8 *buf, int len, char ch) {
-	int i;
-	for (i = 0; i < len; i++) {
-		if (buf[i] != ch) {
-			break;
-		}
-	}
-	return i;
-}
-
 static int getid(char ch) {
 	const char *keys = "[]<>+-,.";
 	const char *cidx = strchr(keys, ch);
 	return cidx ? cidx - keys + 1 : 0;
 }
 
+/* New IL uplift bf */
+#define BF_ADDR_SIZE  64
+#define BF_ALIGN_SIZE 8
+
+struct bf_stack_t {
+	ut64 stack[32];
+	int sp;
```

Comment:
`size_t` is better type (implied that the value represent a size or an index).

Reply:
`sp` or the element of stack ?